src/XeroPHP/Remote/Query.php

```diff
@@ -34,20 +50,65 @@ public function from($class) {
         return $this;
     }
 
-    public function where() {
-        $args = func_get_args();
```

Comment:
I think the reason I did this in the first place was so you could explicitly pass ```null```